src/functions/functionalConfig.ts

```diff
@@ -43,17 +43,17 @@ export const functionConfig: { [key: string]: NonEmptyArray<IApiAccess> } = {
     },
     {
       verbs: ["GET", "OPTIONS"],
-      path: "test-types/*",
+      path: "test-types",
     },
   ],
   "TestResult.CreateContingency": [
     {
       verbs: ["POST", "OPTIONS"],
-      path: "test-result/*",
+      path: "test-results/*",
     },
     {
       verbs: ["GET", "OPTIONS"],
-      path: "test-types/*",
+      path: "test-types",
```

Comment:
you need both test-types and test-types/*

Reply:
Think Nathan tested it and it worked fine

Reply:
test-types doesn't have any endpoints under test-types/

Reply:
it does

Reply:
https://api.develop.cvs.dvsacloud.uk/develop/test-types/:id